src/test/common/steps/navigation.js

```diff
@@ -28,6 +29,7 @@ const EMAIL_ADDRESS_PAGE = 'email address'
 const SEND_CODE_PAGE = 'send code'
 const ENTER_CODE_PAGE = 'enter code'
 const CHECK_PAGE = 'check details'
+const ADD_CHILDREN_DOB_PAGE = 'add your childrens dates of birth'
```

Comment:
Same as above comment `ADD_CHILDREN_DOB_PAGE` -> `ENTER_CHILDREN_DOB_PAGE`

Reply:
Yes, this will be done in another PR, there's a separate sub-task for it now.